job_templates/sag_tf/config_fed_client.conf

```diff
@@ -53,4 +56,14 @@ components = [
       script = "python3 custom/{app_script}  {app_config} "
     }
   }
+  {
+    id = "from_nvflare"
+    path = "nvflare.app_opt.tf.params_converter.NumpyToKerasModelParamsConverter"
+    args {}
+  }
+  {
+    id = "to_nvflare"
+    path = "nvflare.app_opt.tf.params_converter.KerasModelToNumpyParamsConverter"
+    args {}
```

Comment:
@ZiyueXu77 this shows how we use this "params_converter" concept to handle input/output from NVFlare side to external script.

I will think of a similar approach in huggingface scenario.

Reply:
Thanks! yes similar can be done